src/themes/components/popover.ts

```diff
@@ -0,0 +1,19 @@
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react';
+import { popoverAnatomy as parts } from '@chakra-ui/anatomy';
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(parts.keys);
+const sizes = {
+  xl: definePartsStyle({
+    content: {
+      width: '6xl',
+      p: 'md',
+    },
+  }),
+  md: definePartsStyle({
```

Comment:
this does not affect SRP popover, right?

Reply:
No because `md` is the size used by default and it takes initial `p` and `width`